src/electron/tools/router.ts

```
@@ -0,0 +1,188 @@
+import { groq } from "../services/groq.js";
+import { agents, type MakePlanResponse } from "./make_plan.js";
+
+const MAX_ITERATIONS = 10;
+const DONE = "DONE";
+const PROMPT_ROUTE_AGENT = (plan: string, context: any) => `
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)